examples/crypto-websockets/client/asyncapi.yaml

```
@@ -0,0 +1,35 @@
+asyncapi: 2.4.0
+info: 
+  title: asyncapicoin client
+  version: 1.0.0
+  description: |
+    This app creates a client that subscribes to the server for the price change.
+servers:
+  websockets:
+    url: ws://localhost:3000
+    protocol: ws
+    x-kind: remote
```

Let's move this property out of the server definition because that's how it will be done in v3.0.0. For now, we can just use our own extension like this:

```suggestion
x-remoteServers:
  - websockets
```

This will change the logic in the `registerAdapters.ts` file.